src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,67 @@
+package lambda.part1.exercise;
+
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise3 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (a, b) -> {
+            return Integer.compare(a.getAge(), b.getAge());
+        });
+
+        assertArrayEquals(new Person[]{
+            new Person("Иван", "Мельников", 20),
+            new Person("Николай", "Зимов", 30),
+            new Person("Алексей", "Доренко", 40),
+            new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByLastNameThenFirstNameUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (a, b) -> {
+            return a.getLastName().equals(b.getLastName())
+                    ? a.getFirstName().compareTo(b.getFirstName())
+                    : a.getLastName().compareTo(b.getLastName());
+        });
```

Comment:
could you use expression lambda instead of statement?

Reply:
Oh, thx, my fault)